docs/src/performance.md

```diff
@@ -200,64 +199,54 @@ the number of state *variables* for the DOFs. That is, in the previous example
 want to be measured (such as I/O callbacks, visualization etc.).
 
-!!! note "Performance index and adaptive mesh refinement"
-    Currently it is not possible to compute a meaningful PID for a simulation
+!!! note "Performance indicators and adaptive mesh refinement"
+    Currently it is not possible to compute meaningful performance indicators for a simulation
     with arbitrary adaptive mesh refinement, since this would require to
     explicitly keep track of the number of DOF updates due to the mesh size
     changing repeatedly. The only way to do this at the moment is by setting the
     analysis interval to the same value as the AMR interval.
 
-### Local, `rhs!`-only PID
-The *local, `rhs!`-only PID* is computed as
+### Local, `rhs!`-only indicator
+The *local, `rhs!`-only indicator* is computed as
 ```math
-\text{PID}_\text{local,\texttt{rhs!}} = \frac{\{\text{accumulated time spent in \texttt{rhs!}}\}}{n_\text{DOFs,local} \cdot n_\text{calls,\texttt{rhs!}}},
+\text{time/DOF/rhs!} = \frac{\{\text{accumulated time spent in \texttt{rhs!}}\}}{n_\text{DOFs,local} \cdot n_\text{calls,\texttt{rhs!}}},
 ```
 where ``n_\text{DOFs,local}`` is the *local* number of DOFs (i.e., on the
 current MPI rank; if doing a serial run, you can just think of this as *the*
 number of DOFs) and ``n_\text{calls,\texttt{rhs!}}`` is the number of times the
-`rhs!` function has been evaluated. Note that for this PID, we measure *only*
+`rhs!` function has been evaluated. Note that for this indicator, we measure *only*
 the time spent in `rhs!`, i.e., by definition all computations outside of `rhs!` - specifically
 all other callbacks and the time integration method - are not taken into account.
 
-The local, `rhs!`-only PID is usually most useful if you do serial
+The local, `rhs!`-only indicator is usually most useful if you do serial
 measurements and are interested in the performance of the implementation of your
 core numerical methods (e.g., when doing performance tuning).
 
-### Walltime PID
-The *walltime PID* is computed as
+### Performance index (PID)
+The *performance index* (PID) is computed as
 ```math
-\text{PID}_\text{walltime} = \frac{\{\text{walltime since last call to \texttt{AnalysisCallback}}\}}{n_\text{DOFs,global} \cdot n_\text{calls,\texttt{rhs!}}},
+\text{PID} = \frac{\{\text{walltime since last call to \texttt{AnalysisCallback}}\} \cdot n_\text{ranks,MPI}}{n_\text{DOFs,global} \cdot n_\text{calls,\texttt{rhs!}}},
 ```
-where ``n_\text{DOFs,global}`` is the *global* number of DOFs (i.e., the sum of
+where ``n_\text{ranks,MPI}`` is the number of MPI ranks used,
+``n_\text{DOFs,global}`` is the *global* number of DOFs (i.e., the sum of
 DOFs over all MPI ranks; if doing a serial run, you can just think of this as *the*
 number of DOFs) and ``n_\text{calls,\texttt{rhs!}}`` is the number of times the
 `rhs!` function has been evaluated since the last call to the `AnalysisCallback`.
 
```

Comment:
```suggestion
`rhs!` function has been evaluated since the last call to the `AnalysisCallback`.
The PID measures everything except the time spent in the `AnalysisCallback` itself -
specifically, all other callbacks and the time integration method itself are included.

```
Maybe?